### ISSUE TITLE
Contributing Guide

### ISSUE THREAD

**Issue by kriskbx** [kriskbx/gitlab-time-tracker#112]
I added a CONTRIBUTING.md with a list of basic functions to test when accepting PRs or writing new features: https://github.com/kriskbx/gitlab-time-tracker/blob/master/CONTRIBUTING.md

This is just a rough sketch. Let's build on that. :)

**Comment by cgdobre** [comment]
Awesome. Great start! I would add at a first glance to run also the mocha tests and check the Travis build result.

We could also create a public group with subgroup that everyone could test against on order to reduce setup time.

Once that is done we may also create some automated tests for the cases you describe

**Comment by kriskbx** [comment]
Good points. I created a public group over here: https://gitlab.com/gtt I will adjust the contributing guide soon, so it's only copying & pasting commands.

**Comment by almereyda** [comment]
It appears https://gitlab.com/gtt does not exist (anymore)?

Looking forward to bringing this to a stable release (#109) with the community.

I'm still using it and it still works and it's still a nice to have, so I'm willing to put a little effort into it, too.

What was your plan with the gitlab.com/gtt group?
